src/Dashboard/ApiControllers/FunctionsController.cs

```diff
@@ -626,6 +627,12 @@ public IHttpActionResult Abort(string instanceQueueName)
 
         private bool? HostInstanceHasHeartbeat(FunctionInstanceSnapshot snapshot)
         {
+            if (snapshot.HeartbeatExpiredTime.HasValue)
+            {
+                var now = DateTime.UtcNow;
```

Comment:
I have a preference for this style. (splitting out side-effecting or non-idempotent operations into their own locals) 

---
In reply to: [90366513](https://github.com/Azure/azure-webjobs-sdk/pull/928#discussion_r90366513) [](ancestors = 90366513)